frontend/resourceadm/hooks/queries/useGetResourceListQuery.ts

```diff
@@ -12,13 +12,17 @@ import { sortResourceListByDateAndMap } from 'resourceadm/utils/mapperUtils';
  *
  * @returns UseQueryResult with a list of resources of Resource
  */
-export const useGetResourceListQuery = (org: string): UseQueryResult<ResourceListItem[]> => {
+export const useGetResourceListQuery = (
+  org: string,
+  disabled?: boolean,
+): UseQueryResult<ResourceListItem[]> => {
   const { getResourceList } = useServicesContext();
 
   return useQuery<ResourceListItem[]>({
     queryKey: [QueryKey.ResourceList, org],
     queryFn: () => getResourceList(org),
     select: (resourceListItems: ResourceListItem[]) =>
       resourceListItems && sortResourceListByDateAndMap(resourceListItems),
+    enabled: !disabled,
```

Comment:
Should we use the enabled?: boolean instead to ensure following the same pattern as other places we is disabling queries? 😊 


```suggestion
  enabled?: boolean,
): UseQueryResult<ResourceListItem[]> => {
  const { getResourceList } = useServicesContext();

  return useQuery<ResourceListItem[]>({
    queryKey: [QueryKey.ResourceList, org],
    queryFn: () => getResourceList(org),
    select: (resourceListItems: ResourceListItem[]) =>
      resourceListItems && sortResourceListByDateAndMap(resourceListItems),
    enabled,
```

Reply:
Would `enabled` prop be correct? The only other place I can find disabling query from prop in this repo, the prop is named `disabled?:`

```
import { useQuery, UseQueryResult } from '@tanstack/react-query';
import { RepoStatus } from 'app-shared/types/RepoStatus';
import { useServicesContext } from 'app-shared/contexts/ServicesContext';
import { QueryKey } from 'app-shared/types/QueryKey';

export const useRepoPullQuery = (
  owner: string,
  app: string,
  disabled?: boolean, // set to true if you want the browser to NOT perform the API call on refresh
): UseQueryResult<RepoStatus> => {
  const { getRepoPull } = useServicesContext();
  return useQuery<RepoStatus>({
    queryKey: [QueryKey.RepoPullData, owner, app],
    queryFn: () => getRepoPull(owner, app),
    enabled: !disabled,
  });
};
```